src/main/java/seedu/address/logic/commands/ApplicationAddCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COMPANY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONTACT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_POSITION;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.ApplicationModel;
+import seedu.address.model.application.Application;
+
+/**
+ * Adds an application to CinternS.
+ */
+public class ApplicationAddCommand extends ApplicationCommand {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an application to CinternS. "
+            + "Parameters: "
+            + PREFIX_COMPANY + "COMPANY"
+            + PREFIX_CONTACT + "CONTACT "
+            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_POSITION + "POSITION "
+            + PREFIX_DATE + "DATE"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_COMPANY + "ByteDance "
+            + PREFIX_CONTACT + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_POSITION + "Software Engineer (Front-End) "
+            + PREFIX_DATE + "09/10/2022";
```

Comment:
This Date example should be 2022-10-09 (yyyy-MM-dd). The input validation is proposed to change in PR #57 .

Reply:
fixed